components/Slider/src/MDCSlider.m

```diff
@@ -602,7 +600,7 @@ - (void)accessibilityDecrement {
   if (self.enabled) {
     CGFloat range = self.maximumValue - self.minimumValue;
     CGFloat adjustmentAmount = kSliderAccessibilityIncrement * range;
-    if (!self.continuous && self.numberOfDiscreteValues > 1) {
+    if (self.numberOfDiscreteValues > 1) {
```

Comment:
here too

Reply:
Done. Please take a look at the Docs and implementation to see if you agree with how I handled the new configurations.